src/components/chart/state-history-chart-line.ts

```diff
@@ -89,6 +89,11 @@ export class StateHistoryChartLine extends LitElement {
 
   private _yAxisMaximumFractionDigits = 0;
 
+  public isVisible(): boolean {
+    const chartBase = this.shadowRoot!.querySelector("ha-chart-base")!;
```

Comment:
We generally do this with the `@query` decorator. See `_baseChart` in `src/components/chart/ha-network-graph.ts`